package.json

```diff
@@ -1,11 +1,34 @@
 {
-  "name": "webcomponents-jsonforms",
-  "private": true,
-  "version": "0.0.1",
+  "name": "jsonforms",
+  "version": "2.0.0-alpha.3",
+  "description": "Generating forms for JSON schema",
```

Comment:
I think we must go through all descriptions in a separate commit and make it consisten